api/types/device_test.go

```diff
@@ -115,8 +115,11 @@ func TestDeviceConversions_toAndFrom(t *testing.T) {
 		UpdateTime:   timestamppb.New(t2),
 		EnrollStatus: devicepb.DeviceEnrollStatus_DEVICE_ENROLL_STATUS_ENROLLED,
 		Credential: &devicepb.DeviceCredential{
-			Id:           "557762f0-4cd4-4b75-aaee-575c57237c0b",
-			PublicKeyDer: []byte("insert public key here"),
+			Id:                    "557762f0-4cd4-4b75-aaee-575c57237c0b",
+			PublicKeyDer:          []byte("insert public key here"),
+			DeviceAttestationType: devicepb.DeviceAttestationType_DEVICE_ATTESTATION_TYPE_TPM,
```

Comment:
why for macOS we have TPM here? I think it would be better to add either test case for other OS or use unspecified.